backend/api/swagger/experiment.swagger.json

```diff
@@ -154,7 +156,7 @@
         ]
       },
       "delete": {
-        "summary": "Delete an experiment.",
+        "summary": "Delete an experiment. Runs and jobs inside the experiment will stay. It is\nrecommended to delete runs and jobs inside the experiment before deleting\nthe experiment in order to avoid unexpected behaviors.",
```

Comment:
Suggested revision:

Deletes an experiment without deleting the experiment's runs and jobs. To avoid unexpected behaviors, delete an experiment's runs and jobs before deleting the experiment.

Reply:
addressed in proto file